backend/routes/admin.py

```
@@ -0,0 +1,836 @@
+            return {"gaps": gaps, "total": len(gaps)}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching content gaps: {str(e)}")
+
+
+# Export endpoints
+@router.get("/export/csv")
+async def export_csv(
+    start_date: Optional[str] = Query(None),
+    end_date: Optional[str] = Query(None),
+    export_type: str = Query("queries", pattern="^(queries|metrics)$"),
+    session: Dict[str, Any] = Depends(require_admin_auth),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)